OloEngine/src/OloEngine/Renderer/IBLCache.cpp

```
@@ -0,0 +1,578 @@
+
+        OLO_CORE_INFO("IBLCache: Initialized with {} bytes cached", s_Stats.CacheSizeBytes);
+    }
+
+    void IBLCache::Shutdown()
+    {
+        s_Initialized = false;
+        OLO_CORE_INFO("IBLCache: Shutdown (Hits: {}, Misses: {}, Saves: {})",
+                      s_Stats.CacheHits, s_Stats.CacheMisses, s_Stats.SaveCount);
+    }
+
+    u64 IBLCache::ComputeHash(const std::string& sourcePath, const IBLConfiguration& config)
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)